model/stream.go

```diff
@@ -41,10 +41,30 @@ type Stream struct {
 	StreamName       string
 }
 
+// IsRegularStream returns whether the stream is a scheduled stream in a lecture hall
+func (s Stream) IsRegularStream() bool {
```

Comment:
I'd call this something like `isNotSelfStream`

Reply:
The name is not ideal. I think it would be even better to invert the logic here and call `isSelfStream` instead. Otherwise it gets confusing when we negate the variable again.